torchci/rockset/prodVersions.json

```diff
@@ -19,14 +19,14 @@
     "last_branch_push": "3d995ac2143586dc",
     "last_successful_jobs": "fd8bc3a2afc989a2",
     "last_successful_workflow": "5d22927dd0b0956b",
-    "master_commit_red_avg": "fbb27c782608d52d",
-    "master_commit_red": "e5003789ace2ca6a",
-    "master_jobs_red_avg": "a017cf67c6a503a5",
-    "master_jobs_red": "739a319fd5133800",
-    "queued_jobs_by_label": "6ddd544c9608d394",
-    "queued_jobs": "3724769f76ad4ecc",
-    "reverts": "235c444d865a5e48",
+    "master_commit_red_avg": "70391ac17f1c45bd",
```

Comment:
how come so many of these need to change for this PR which doesn’t touch all of them?

Reply:
probably weird spacing changes; I confirmed there is no material difference.